go/cmd/vtctldclient/command/workflows.go

```diff
@@ -184,10 +355,32 @@ func init() {
 	Workflow.PersistentFlags().StringVarP(&workflowOptions.Keyspace, "keyspace", "k", "", "Keyspace context for the workflow (required)")
 	Workflow.MarkPersistentFlagRequired("keyspace")
 	Root.AddCommand(Workflow)
+
+	WorkflowDelete.Flags().StringVarP(&workflowDeleteOptions.Workflow, "workflow", "w", "", "The workflow you want to update (required)")
```

Comment:
so Persistent flags let you define the flag once on the parent, then you don't need to keep redefining it here